share/spice/nobel_prize/nobel_prize.js

```diff
@@ -0,0 +1,141 @@
+(function (env) {
+    "use strict";
+    env.ddg_spice_nobel_prize = function(api_result) {
+
+        // Validate the response (customize for your Spice)
+        if (!api_result || !api_result.prizes || api_result.prizes.length == 0) {
+            return Spice.failed('NobelPrize');
+        }
+
+        function getURLSurname(surname) {
+            surname = surname.toLowerCase();
+            surname = surname.replace(/(?!-)[\W]/g, '');   
+            return surname;
+        }
+
+        function getSubtitle(laureates) {
+            var numLaureates = laureates.length,
+                subtitle = '';
+            
+            if (numLaureates > 1) {
+                    subtitle = "Shared with " + laureates[1].firstname;
+                    subtitle += (numLaureates == 2 ? "." : " and " + (numLaureates - 2) + " more.");
+                } else 
+                    subtitle = "Shared by none.";
+            
+            return subtitle;
+        }
+
+        function getSearchTerm() {
+            var script = $('[src*="/js/spice/nobel_prize/"]')[0],
+            source = $(script).attr("src"),
+            query = source.match(/nobel_prize\/([^\/]+)/)[1],
+            decodedQuery = decodeURIComponent(query),
+            year = decodedQuery.match(/(\d{4})/) && decodedQuery.match(/(\d{4})/)[1],
+            category = decodedQuery.match(/(physics|chemistry|medicine|peace|literature|economics)/) 
```

Comment:
You shouldn't need regex here. If the array is two items, one will just be numbers, the other must be the category you sent to the api. If it isn't a number it must be a category.